doc/internal/ReleaseInstructions.md

```diff
@@ -275,6 +276,7 @@ git checkout master
 git pull --ff-only upstream master
 
 # Example:
```

Comment:
I was comparing it with the Example in line 212, where you have given an example on the same line where Example: is mentioned.